metis-enrichment/metis-enrichment-rest/src/main/java/eu/europeana/enrichment/rest/EnrichmentController.java

```diff
@@ -33,115 +30,98 @@
 @Controller
 public class EnrichmentController {
 
-  private final Converter converter;
-  private final Enricher enricher;
+  private final EnrichmentService enrichmentService;
 
   /**
    * Autowired constructor.
    *
-   * @param enricher class that handles enrichment functionality
-   * @param converter utility class for conversion between different classes
+   * @param enrichmentService the service that contains logic for processing entities
    */
   @Autowired
-  public EnrichmentController(Enricher enricher, Converter converter) {
-    this.converter = converter;
-    this.enricher = enricher;
+  public EnrichmentController(EnrichmentService enrichmentService) {
+    this.enrichmentService = enrichmentService;
   }
 
   /**
-   * Get an enrichment by URI (rdf:about or owl:sameAs/skos:exactMatch).
+   * Get an enrichment by URI, might match owl:sameAs.
    *
-   * @param uri The URI to retrieve
+   * @param uri The URI to check for match
    * @return the structured result of the enrichment
-   * @throws EnrichmentException if an exception occurred during enrichment
    */
-  @GetMapping(value = RestEndpoints.ENRICHMENT_BYURI, produces = {MediaType.APPLICATION_JSON_VALUE,
+  @GetMapping(value = RestEndpoints.ENRICH_CODEURI_OR_OWLSAMEAS, produces = {
+      MediaType.APPLICATION_JSON_VALUE,
       MediaType.APPLICATION_XML_VALUE})
-  @ApiOperation(value = "Retrieve an entity by rdf:about or owl:sameAs/skos:exactMatch", response = EnrichmentBase.class)
+  @ApiOperation(value = "Get an enrichment by URI, might match owl:sameAs", response = EnrichmentBase.class)
```

Comment:
Why change these comments? I thought the original was clearer. It's still true, right? Also below.

Reply:
What was there makes sense generally but if you think about it maybe not exactly..
The db has the values:
- codeUri
- owlSameAs

I do not know how exactly those fields were filled in exactly.
And whoever is searching the db is using a uri(we don't know from what field that comes from), with the scope of getting a match from the codeUri or owlSameAs field in the db..
What do you think the comments should be in this case?

Reply:
What threw me off is that you specifically name owl:sameAs, but no longer rdf:about. Seems oddly matched this way.

Admittedly, this is not the most vital comment in the world, so I'm fine to leave it as it is (i.e. your new phrasing). But if you feel like changing it, perhaps we can have the best of both worlds, and just put the specific field names it matches in parentheses by way of explanation?

Reply:
Updated the documentation